coriolis/tasks/replica_tasks.py

```diff
@@ -39,17 +43,14 @@ def _check_ensure_volumes_info_ordering(export_info, volumes_info):
         if not matching_volumes:
             raise exception.InvalidActionTasksExecutionState(
                 "Could not find source disk '%s' (ID '%s') in Replica "
-                "volumes info: %s" % (disk, disk_id, volumes_info))
+                "volumes info: %s" % (disk, disk_id, vol_info_cpy))
         elif len(matching_volumes) > 1:
             raise exception.InvalidActionTasksExecutionState(
-                "Multiple disks with ID '%s' foind in Replica "
-                "volumes info: %s" % (disk_id, volumes_info))
+                "Multiple disks with ID '%s' found in Replica "
+                "volumes info: %s" % (disk_id, vol_info_cpy))
```

Comment:
Same here, keep the message, change to `LOG.error()`;
Raise exception with message:
```python
raise exception.InvalidActionTasksExecutionState(
    "Multiple disks with ID '%s' found in Replica volumes info. Please check that the instance doesn't have the "
    "same volume attached twice, or whether there's a UUID collision between its disks. Check the logs "
    "for more information." % disk_id)
```